rules/configuration.smk

```diff
@@ -40,12 +40,14 @@ if 'condition' in samples:
                 f' condition {condition} and assembly {assembly}'
 
 
-# cut off trailing slashes and make absolute path
+# make absolute paths, cut off trailing slashes and nestle default directories in result_dir
 for key, value in config.items():
     if '_dir' in key:
         if key in ['result_dir', 'genome_dir', 'rule_dir']:
             value = os.path.abspath(value)
         config[key] = re.split("\/$", value)[0]
+        if key in ['sra_dir', 'fastq_dir', 'trimmed_dir', 'qc_dir', 'dedup_dir'] and key not in user_dirs:
+            config[key] = os.path.join(config['result_dir'], config[key])
```

Comment:
So what happens if for instance sra_dir goes through the loop before result_dir? That will give us some bad parsing here I guess? I am pretty sure these loops should be split!

Reply:
you are right!